src/bacnet/basic/object/csv.c

```diff
@@ -181,13 +181,13 @@ bool CharacterString_Value_Valid_Instance(uint32_t object_instance)
     unsigned index = 0; /* offset from instance lookup */
```

Comment:
it is legal in ansi-c to: 'unsigned index = CharacterString_Value_Instance_To_Index(object_instance);'